src/olympia/urls.py

```diff
@@ -76,9 +76,12 @@
         r'^addons/versions/(\d+)/?$',
         lambda r, id: redirect('addons.versions', id, permanent=True),
     ),
-    # Legacy redirect. Requires a view to get extra data not provided in URL.
+    # Legacy redirect. Doesn't receive the addon id/slug so it can't live with
+    # the other version views that do.
     re_path(
-        r'^versions/updateInfo/(?P<version_id>\d+)', version_views.update_info_redirect
+        r'^versions/updateInfo/(?P<version_id>\d+)',
+        version_views.update_info_redirect,
+        name='addons.versions.update_info_redirect',
```

Comment:
I meant the entire thing is repeated - the name (`'addons.versions.update_info_redirect'`), the path, and that it's for the `update_info_redirect` view.  

I guess I should have added the comment next to the `versions.url` lines because as far as I can see that's the incorrect one (no need to redirect from a url that already contains to the slug)

Reply:
Ah right, that's a leftover from an attempt I made at moving this that I eventually abandonned. You're right that the incorrect one is the `versions.url` one, I'll remove it